_docs/contracts.md

```diff
@@ -113,6 +113,11 @@ Restrictions for libraries in comparison to contracts:
 
 (these might be lifted at a later point)
 
+Common pitfalls for libraries
+
+* The value for `msg.sender` will be that of the contract which is calling the library function.  (If A calls contract B which internally calls library C, then within the function call of library C, `msg.sender` will be the address of contract B).
```

Comment:
I would say that it is important to explain the reason for that, here:

The expression `LibraryName.functionName()` performs an external function call, which maps to a real EVM call just like `otherContract.functionName()` or `this.functionName()` and thus extends the call depth by one (limited to 1024), stores the caller as `msg.sender`, it is possible to transfer ether together with this call (using `.value(x)`), creates a completely new memory context, and thus memory types will be copied and cannot be passed by reference, and so on. The semantics of that are planned to be explained in the yet to be written page about the Ethereum Virtual Machine.